chsdi/templates/htmlpopup/cadastralwebmap.mako

```diff
@@ -6,7 +6,7 @@
 <%def name="table_body(c, lang)">
     <%
       from chsdi.models.vector import getScale
-      from chsdi.lib.validation.features import IdentifyServiceValidation
+      from chsdi.lib.validation.identify import IdentifyServiceValidation
```

Comment:
No. Just test that this mako still works as expected. Do not change IdentifyServiceValidation in identify validation as it's already used elsewhere. Only this mako used the old and deprecated version - but it might be that the new version now breaks the mako.